travis-run-tests.sh

```diff
@@ -1,8 +1,10 @@
 #! /usr/bin/env bash
 
+set -e
+
 current_version=`python setup.py --version`
 
-if ! git diff-index --quiet master openfisca_web_api
+if [[ $(git --no-pager diff --ignore-blank-lines  master openfisca_web_api) ]]
```

Comment:
You're not testing the same thing as before 🤔
Can't you keep using the exit value of [`diff-index`](https://git-scm.com/docs/git-diff-index) rather than evaluating the emptiness of the string returned by `diff`? It's better to rely on [plumbing than porcelain](http://stackoverflow.com/a/6976506).